src/bin/cratesfyi.rs

```diff
@@ -30,15 +47,23 @@ pub fn main() {
         if !backtrace.is_empty() {
             eprintln!("\nStack backtrace:\n{}", backtrace);
         }
+
+        capture_anyhow(&err);
```

Comment:
Hmm, now that I think about it more I think @Nemo157 had a good point in https://github.com/rust-lang/docs.rs/pull/1152/files#r515400948 - any top-level errors will show up immediately after you run the command, so they'll be something we (me and pietro) can interact with in real time; there's no need to log them. The errors that are important are the ones from `ctry!` and `cexpect!`, which come when the daemon is running unattended. So I don't think we should send an event here.
```suggestion
```